CODEOWNERS

```diff
@@ -115,6 +115,7 @@
 /drivers/dma/*sam0*                       @Sizurka
 /drivers/ethernet/                        @jukkar @tbursztyka @pfalcon
 /drivers/flash/                           @nashif
+drivers/flash/*native_posix*              @aescolar @vanwinkeljan
```

Comment:
first '/' missing?
(I already get all `*native_posix*` drivers, so no need to have me in the line)